vlasovsolver/cpu_trans_map_amr.cpp

```diff
@@ -1316,18 +1544,33 @@ void prepareSeedIdsAndPencils(const dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Ge
    if(printPencils) MPI_Comm_rank(MPI_COMM_WORLD,&myRank);
 
    const vector<CellID>& localCells = getLocalCells();
-   vector<CellID> localPropagatedCells;
+   vector<CellID> PropagatedCells;
    // Figure out which spatial cells are translated,
    // result independent of particle species.
-   for (size_t c=0; c<localCells.size(); ++c) {
-      if (do_translate_cell(mpiGrid[localCells[c]])) {
-         localPropagatedCells.push_back(localCells[c]);
+   if (P::vlasovSolverLocalTranslate) {
+      // Sets already include check for do_translate_cell 
+      switch (dimension) {
+         case 0:
+            PropagatedCells.assign(LocalSet_x.begin(), LocalSet_x.end());
+            break;
+         case 1:
+            PropagatedCells.assign(LocalSet_y.begin(), LocalSet_y.end());
+            break;
+         case 2:
+            PropagatedCells.assign(LocalSet_z.begin(), LocalSet_z.end());
+            break;
```

Comment:
Just so I don't break this one day, please add a default? :)

Reply:
Roger wilco